File: ruby/lonely_integer.rb
def lonelyinteger(test_case)
  test_case.each do |num|
    return num if test_case.count(num) == 1
  end
end
_ = gets.strip.to_i
test_case = gets.strip.split(' ').map!(&:to_i)
print lonelyinteger(test_case)

File: python/lonely-integer.py
def lonelyinteger(nums):
    for i in range(len(nums)):
        if nums.count(nums[i]) == 1:
            return nums[i]


if __name__ == '__main__':
    a = input()
    nums = map(int, raw_input().strip().split(" "))
    print(lonelyinteger(nums))
